submissions/nadsatt/friends-app/main.js

```diff
@@ -0,0 +1,51 @@
+import { ApiService } from './modules/api-service.js';
+import { UserService } from './modules/user-service.js';
+
+import { LoadingPage } from './modules/loading-page.js';
+import { Header } from './modules/header.js';
+import { UserCardList } from './modules/user-card-list.js';
+import { PageLinkList } from './modules/page-link-list.js';
+import { FilterGroup } from './modules/filter-group.js';
+
+class Program {
+
+    constructor(){
+        this.defineServices();
+        this.defineComponents();
+    }
+
+    defineServices(){
+        this.apiService = new ApiService();
```

Comment:
You should declare all such variables in your constructor.

In such a way it will be understandable what your class have.

Also, in general, initialization also can be done in constructor, if it's simple enough.

In other case, you smearing logic of your application.